recipes/r-geometa/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set version = '0.6-6' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-geometa
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/geometa_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/geometa/geometa_{{ version }}.tar.gz
+  sha256: 1d63a51eb50c457248ae4a24550ca158c0f78c169c65dc67d83283f90a4b0220
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-r6
+    - r-xml
+    - r-httr
+    - r-jsonlite
+    - r-keyring
+  run:
+    - r-base
+    - r-r6
+    - r-xml
+    - r-httr
+    - r-jsonlite
+    - r-keyring
+
+test:
+  commands:
+    - $R -e "library('geometa')"           # [not win]
+    - "\"%R%\" -e \"library('geometa')\""  # [win]
+
+about:
+  home: https://github.com/eblondel/geometa/wiki
+  license: MIT
+  summary: Provides facilities to handle reading and writing of geographic metadata defined with
+    OGC/ISO 19115, 11119 and 19110 geographic information metadata standards, and encoded
+    using the ISO 19139 (XML) standard. It includes also a facility to check the validity
+    of ISO 19139 XML encoded metadata.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
```

as stated by the cran meta data
```suggestion
    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
    - LICENSE
```